#!/usr/bin/env python
PACKAGE = "amaze_autonomy"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("option_a",    bool_t,    0, "Some Description", False)

exit(gen.generate(PACKAGE, "Autonomy", "autonomy"))
